main.py

```diff
@@ -70,6 +72,38 @@ def report(ctx, output):
     exporter.generate(project, 'report.html')
     pass
 
+@cli.command()
+@click.option('--version-file', default='version.csv', help='CSV Version file')
+@click.option('--overwrite', default=False, help='Overwrite database')
+@click.pass_context
+def import_csv(ctx, version_file, overwrite):
+    """Import CSV file into tables"""
+    # Overwrite option
```

Comment:
All this stuff should be moved into a dedicated package "importers" and you should create a dedicated object. Too long for main and not consistent with the rest of the code.